examples/java-demo/src/main/java/org/example/Main/Main.java

```diff
@@ -0,0 +1,66 @@
+package org.example.Main;
+
+import com.carta.excel.AppendOnlySheetWriter;
+import com.carta.exscalabur.DataCell;
+import com.carta.exscalabur.DataRow;
+import com.carta.exscalabur.Exscalabur;
+import com.carta.yaml.YamlEntry;
+import com.carta.yaml.YamlReader;
+import org.apache.commons.compress.utils.Lists;
+import scala.collection.JavaConverters;
+import scala.collection.immutable.Map;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        new App().run();
+    }
+}
+
+class App {
+    public void run() {
+        final ClassLoader classLoader = this.getClass().getClassLoader();
+        final String templatePath = classLoader.getResource("excel/template.xlsx").getFile();
+        final String schemaPath = classLoader.getResource("yaml/schema.yaml").getFile();
+
+        final ArrayList<String> templates = Lists.newArrayList();
+        templates.add(templatePath);
+
+        final Map<String, YamlEntry> schemaDefinition = YamlReader.apply().parse(schemaPath);
+
+        final Exscalabur sword = Exscalabur.apply(
+                JavaConverters.collectionAsScalaIterable(templates),
```

Comment:
Should we provide an API for Java users or an implicit conversion from Lists to Seq?